python/baseline/utils.py

```diff
@@ -19,6 +19,7 @@
 import numpy as np
 import addons
 import collections
+import yaml
```

Comment:
yaml is an optional dependency in baseline so this import should be inside the yaml function.